app/controllers/branding_groups_controller.rb

```diff
@@ -90,6 +97,16 @@ def destroy
     end
   end
 
+  # helper to merge in the list of curators into the :users parameter
+  # will prevent curator from removing themselves from the collection
+  def self.merge_curator_params(curator_list, collection, user)
+    curators = curator_list.split(',').map(&:strip)
+    users = curators.map { |email| User.find_by(email: email) }.compact
+    # ensure current user cannot remove accidentally remove themselves from the list
+    users << user if collection.present? && collection.users.include?(user) && !users.include?(user)
```

Comment:
Can this be more succinctly expressed via [safe navigation operator](https://stackoverflow.com/questions/36812647/what-does-ampersand-dot-mean-in-ruby)?

```suggestion
    users << user if collection&.users.include?(user) && !users.include?(user)
```

Reply:
It would have to be `collection&.users&.include?(user)` to prevent a `NoMethodError` if the collection isn't present, and I'm not a fan of multiple nil-safe operators in a row.  I find them to be somewhat obtuse.  I could do this, however:

```
curators = curator_list.split(',').map(&:strip)
users = curators.map { |email| User.find_by(email: email) }.compact
return users if collection.nil?

# ensure current user cannot accidentally remove themselves from the list if this is an update
users << user if collection.users.include?(user) && !users.include?(user)
users
```

Reply:
70eb474